src/stdlib_hashmap_wrappers.f90

```diff
@@ -206,9 +209,10 @@ subroutine get_char_key( key, value )
             key_as_char = key_size
         case(2)
             if ( iand( key_size, 1_int64 ) > 0 ) then
-                error stop "Internal Error at stdlib_hashmaps:&
-                           & System uses 2 bytes per character, so&
-                           & key_size can't be an odd number"
+                error stop module_name // " % " // procedure // &
+                          ": Internal Error at stdlib_hashmaps: " // &
+                           "System uses 2 bytes per character, so " // &
+                           "key_size can't be an odd number"
             end if
             key_as_char = ishft( key_size, -1 )
         case(4)
```

Comment:
The other cases below could benefit from the same improvements.

Reply:
I believe I have fixed the two cases you refer to.